front-end/config/environment.js

```diff
@@ -21,6 +21,22 @@ module.exports = function (environment) {
       // Here you can pass flags/options to your application instance
       // when it is created
     },
+    'mapbox-gl': {
+      accessToken:
+        // should be set in doppler?
+        'pk.eyJ1Ijoic3licmVuZGIiLCJhIjoiY2txdGtvYWJ3MDBiNjJ1bW1qc2Z1NDJqZyJ9.pU0f9LsEAo44NXmktpdnYA',
```

Comment:
I'm not familiar with doppler, but be very careful with adding API tokens on git. There are bots scraping GitHub for anything that looks like an API token.

I'm actually not sure what the proper way to do this would be, since the client side does need access to that token. It's not like backend where you can just set it as an environment variable :/

Reply:
Removed this as it is only used for mapbox